orchestrator/src/cli/provider/aws.rs

```diff
@@ -3,17 +3,12 @@ use serde::Serialize;
 
 /// Parameters used to config AWS.
 #[derive(Debug, Clone, Args, Serialize)]
-#[group(requires_all = ["aws_region"])]
 pub struct AWSConfigCliArgs {
     /// Use this flag to enable AWS provider.
     #[arg(long)]
     pub aws: bool,
-
-    /// The region.
-    #[arg(env = "AWS_REGION", long)]
-    pub aws_region: String,
-
-    /// The region.
-    #[arg(env = "MADARA_ORCHESTRATOR_AWS_PREFIX", long, default_value = "madara-orchestrator")]
-    pub aws_prefix: String,
+    /// The prefix value.
+    /// And added to the start of each resource name if available
+    #[arg(env = "MADARA_ORCHESTRATOR_AWS_PREFIX", long, default_value = None)]
+    pub aws_prefix: Option<String>,
```

Comment:
```suggestion
    #[arg(env = "MADARA_ORCHESTRATOR_AWS_PREFIX", long, default_value = "mo")]
    pub aws_prefix: String,
```
since at anypoint we can have none if other parameters are given as name

Reply:
I believe this was discussed offline, 
we are following this structure because of clap,
we can take up refactoring clap in next few PRs.